plugins/utils/mini.nix

```diff
@@ -1,51 +1,90 @@
 {
   lib,
-  helpers,
-  config,
-  pkgs,
   ...
 }:
-with lib;
-let
-  cfg = config.plugins.mini;
-in
-{
-  options.plugins.mini = {
-    enable = mkEnableOption "mini.nvim";
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "mini";
+  originalName = "mini.nvim";
 
-    package = lib.mkPackageOption pkgs "mini.nvim" {
-      default = [
-        "vimPlugins"
-        "mini-nvim"
-      ];
-    };
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  package = "mini-nvim";
 
-    modules = mkOption {
-      type = with types; attrsOf attrs;
+  extraOptions = {
+    modules = lib.mkOption {
+      type = with lib.types; attrsOf (attrsOf anything);
       default = { };
       description = ''
         Enable and configure the mini modules.
-        The keys are the names of the modules (without the `mini.` prefix).
-        The value is an attrs of configuration options for the module.
-        Leave the attrs empty to use the module's default configuration.
+
+        The attr name represent mini's module names, without the `"mini."` prefix.
+        The attr value is an attrset of options provided to the module's `setup` function.
+
+        See the [plugin documentation] for available modules to configure:
+
+        [plugin documentation]: https://github.com/echasnovski/mini.nvim?tab=readme-ov-file#modules.
```

Comment:
Sorry, I think this was my fault: (trailing `.`)
```suggestion
        [plugin documentation]: https://github.com/echasnovski/mini.nvim?tab=readme-ov-file#modules
```

Reply:
Hmm.. it looked fine though strange enough
![image](https://github.com/user-attachments/assets/7fcef760-a2a9-4441-8ba4-e764b0f4e70d)

Reply:
It renders fine, but will link to the wrong place.

Reply:
Yeah, i deleted when I realized that :P